pup-0008.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Python Code Style and Tools
+author: David Davis
+created: 8-Jun-2019
+status:
+version: 1
+---
+
+## Summary
+
+PUP-8 proposes the adoption of an official Python code style and set of tools for the Pulp
+community.
+
+### Background
+
+While most Pulp projects adhere to PEP 8[0], there is still room for different code styles to emerge
+and different toolsets to be used to enforce style for Python code. This PUP seeks to come to a
+common agreement about which style and tools we use. Specifically, this PUP proposes adoption of
+black[1] along with pydocstyle[2] as the official toolset for formatting and linting Python code.
+
+## Motivation
+
+The Pulp ecosystem contains numerous different coding styles that make it difficult to switch from
```

Comment:
FWIW, I understand that this problem potentially exists or will be more prominent at some point. I don't feel/see it right now.